app/models/sat/catalogo/base.rb

```diff
@@ -4,10 +4,11 @@ class Base < ActiveRecord::Base
       self.abstract_class = true
 
       def self.search(keyword)
+        keyword.empty? ? limite = nil : limite = 20
         conditions = searchable_columns.map do |column|
           "#{column} like :keyword"
         end.join(' OR ')
-        where(conditions, keyword: "%#{keyword}%").limit(20)
+        where(conditions, keyword: "%#{keyword}%").limit(limite)
```

Comment:
Why not only do:  `where(conditions, keyword: "%#{keyword}%")&.limit(20)` & remove line 7